apps/api/src/app/layouts/layouts.controller.ts

```diff
@@ -99,34 +102,9 @@ export class LayoutsController {
     };
   }
 
-  @Get('')
+  @Get()
   @ExternalApiAccessible()
-  @ApiQuery({
-    name: 'page',
-    type: Number,
-    description: 'Number of page for the pagination',
-    required: false,
-  })
-  @ApiQuery({
-    name: 'pageSize',
-    type: Number,
-    description: 'Size of page for the pagination',
-    required: false,
-  })
-  @ApiQuery({
-    name: 'sortBy',
-    type: String,
-    description: 'Sort field. Currently only supported `createdAt`',
-    required: false,
-  })
-  @ApiQuery({
-    name: 'orderBy',
-    type: Number,
-    description: 'Direction of the sorting query param. Either ascending (1) or descending (-1)',
-    required: false,
-  })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** why did we remove this one?

Reply:
moved into the internal object, is was scattered